debian/tests/integration

```diff
@@ -7,7 +7,7 @@ dpkg-source --before-build .
 # prepare the environment
 ./debian/rules dh_configure
 export GOPATH="$PWD/.gopath"
-export DOCKER_GITCOMMIT="$(./debian/helpers/gitcommit.sh)"
+#export DOCKER_GITCOMMIT="$(./debian/helpers/gitcommit.sh)"
```

Comment:
If this ends up breaking when run, it can probably just be something like `DOCKER_GITCOMMIT=foo`, honestly (since it won't make much difference for the tests).

Reply:
ack